sql/core/src/main/scala/org/apache/spark/sql/execution/SparkQl.scala

```diff
@@ -370,13 +370,33 @@ private[sql] class SparkQl(conf: ParserConf = SimpleParserConf()) extends Cataly
       case Token("TOK_CLEARCACHE", Nil) =>
         ClearCacheCommand
 
+      // SHOW TABLES [(IN|FROM) database_name] ['identifier_with_wildcards']
```

Comment:
@andrewor14 Hi Andrew, is it ok if i have a very minor change in this file just to make sure compilation goes through. Or it will still cause issue with merging ?

ShowTablesCommand(databaseName, None) => we introduced a new parameter to the command.